extensions/wikia/WikiaMaps/controllers/WikiaMapsSpecialController.class.php

```diff
@@ -42,7 +42,6 @@ public function __construct( $name = null, $restriction = '', $listed = true, $f
 	 * Wikia Maps special page
 	 */
 	public function index() {
-		$this->wg->SuppressPageHeader = true;
 		$this->wg->out->setHTMLTitle( wfMessage( 'wikia-interactive-maps-title' )->escaped() );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)